empatia.tf

```diff
@@ -0,0 +1,141 @@
+resource "aws_iam_user" "empatia" {
+  name = "empatia"
+}
+
+resource "aws_iam_access_key" "empatia" {
+  user = aws_iam_user.empatia.name
+}
+
+resource "aws_route53_zone" "empatia" {
+  name = "bankempatii.pl"
+}
+
+resource "aws_route53_record" "a_empatia" {
+  zone_id = aws_route53_zone.empatia.zone_id
+  name    = aws_route53_zone.empatia.name
+  type    = "A"
+  alias {
+    evaluate_target_health = false
+    name                   = "d6cxmr8432pje.cloudfront.net"
+    zone_id                = "Z2FDTNDATAQYW2"
+  }
+}
+
+resource "aws_route53_record" "a_www_empatia" {
+  zone_id = aws_route53_zone.empatia.zone_id
+  name    = "www.${aws_route53_zone.empatia.name}"
+  type    = "A"
+  ttl     = "300"
+  records = [
+    "176.31.162.42",
+  ]
+}
+
+resource "aws_route53_record" "mx_empatia" {
+  zone_id = aws_route53_zone.empatia.zone_id
+  name    = aws_route53_zone.empatia.name
+  type    = "MX"
+  ttl     = "300"
+  records = [
+    "1 redirect.ovh.net",
+  ]
+}
+
+resource "aws_route53_record" "mx_www_empatia" {
+  zone_id = aws_route53_zone.empatia.zone_id
+  name    = "www.${aws_route53_zone.empatia.name}"
+  type    = "MX"
+  ttl     = "300"
+  records = [
+    "1 redirect.ovh.net",
+  ]
+}
+
+resource "aws_route53_record" "ns_empatia" {
+  zone_id = aws_route53_zone.empatia.zone_id
+  name    = aws_route53_zone.empatia.name
+  type    = "NS"
+  ttl     = "172800"
+  records = [
+    "ns-214.awsdns-26.com",
+    "ns-1083.awsdns-07.org",
+    "ns-1596.awsdns-07.co.uk",
+    "ns-854.awsdns-42.net",
+  ]
+}
+
+resource "aws_route53_record" "soa_empatia" {
+  zone_id = aws_route53_zone.empatia.zone_id
+  name    = aws_route53_zone.empatia.name
+  type    = "SOA"
+  ttl     = "900"
+  records = [
+    "ns-1596.awsdns-07.co.uk. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 86400",
+  ]
+}
+
+resource "aws_route53_record" "cname_empatia" {
```

Comment:
that should be removed here, as that's a domain verification record from AWS ACM and it will be recreated in `./ssl_certificate`